1-the-basics/css/styles.css

```
@@ -0,0 +1,70 @@
+header{
```

```suggestion
header {
```
and use classnames over tag names to not force your HTML to match your CSS